### PR TITLE
Added feed to footers, many clean-ups

### LINKED ISSUE #70: Add blogs feeds to all pages

### LINKED ISSUE #78: Update homepage contact-us links
they link to pages.u.c, they should link to the current site.

### LINKED ISSUE #81: new global nav missing from japanese site
## Summary

we designed a global nav for the Japanese website and I'm sure we built it but now it's showing the old one.

## Screenshot

screenshot 2019-01-31 at 15 17 33

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- added the footer feed include
- fixed up contact-us
- fixed links
- updated plans-and-pricing and support pages to follow the correct hierarchy and look in copy doc better
- removed u.c redirects
- move main-content id to base.html
- added copy docs to all files
- updated copy docs to look like u.c ones
- commented out blog from the nav
- removed taleo js

## QA

1. download this branch
2. look at all pages

## Issue / Card

Fixes #78 
Fixes #70 
Fixes #81

### CODE REVIEW

**Comment by deadlight** [comment]
I'm not sure if it was introduced in these changes but the homepage hero is  a bit off on mobile:
image

**Comment by deadlight** [comment]
On /openstack, in the contextual footer there's the string "日本語ウェブサイトからのフィード" which is actually "Feed from Japanese website" which was mistakenly translated.

File: templates/_base/footer.html [hunk 1]
```diff
@@ -25,30 +25,6 @@
   </div>
 </footer>
 
-<script src="/static/js/lib/jquery-2.2.2.min.js"></script>
-<script>
-
-// Production feed - swap out when CORS enabled https://insights.ubuntu.com/group/cloud-and-server/feed/
-// var xml_feed = "https://insights.ubuntu.com/group/cloud-and-server/feed/";"
-
-// Temp feed from Peter's comm site
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/_base/footer.html]: of course!... I forgot

**Comment by pmahnke** [comment]
@anasereijo the download positive and neutral buttons are meant to suggest to download the LTS over the other version

**Comment by anasereijo** [comment]
@pmahnke thanks for making the changes. 
About the download buttons, I think it works better as it was before. 👍

**Comment by shivjoshi1996** [comment]
@clagom This demo includes all of the new Japanese pages

**Comment by clagom** [comment]
Should I open a separated issue for these? All about breaking the sentences on the right points
Just on the homepage

screenshot 2019-02-04 at 11 01 53

screenshot 2019-02-04 at 11 08 54

screenshot 2019-02-04 at 11 10 00

screenshot 2019-02-04 at 11 10 35

**Comment by clagom** [comment]
On another matter, I think the active item should be highlighted in the menu
e.g. OpenStack page, the item in the menu should have the Vanilla underline below?
screenshot 2019-02-04 at 11 19 00
@deadlight

**Pull request merged**